cmd/operator-sdk/generate/csv.go

```diff
@@ -53,9 +53,8 @@ for the operator. This file is used to publish the operator to the OLM Catalog.
 
 A CSV semantic version is supplied via the --csv-version flag. If your operator
 has already generated a CSV manifest you want to use as a base, supply its
-version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.
+version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.`,
 
-Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config.yaml`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			// The CSV generator assumes that the deploy and pkg directories are
 			// present at runtime, so this command must be run in a project's root.
```

Comment:
(Can't comment on the line just below this)
@estroz This isn't true anymore since the inputs and output dir are configurable.
So I'm going to remove the requirement for running in project root i.e check `build/Dockerfile` exists.
I'll need this for the CSV generator to work on kubebuilder's layout plus other users can have non standard layouts.